### ISSUE TITLE
Tables in appendix don't render properly when knitting to .docx

### ISSUE THREAD

**Issue by kristabh** [manybabies/mb1b-analysis-public#44]
The tables look fine when knitting to .pdf, but we need to also knit to .docx for typesetting. @jkosie would you be able to figure this one out?

**Comment by kristabh** [comment]
Oops, opened this issue in the wrong repo.  Sorry!  This is an mb1b issue, here: https://github.com/manybabies/mb1b-analysis-public/issues/44

Closing this one.

**Comment by kristabh** [comment]
Accidentally had originally opened this issue in the mb1 repo - deleted there now.  @kachergis mentions:

"I've run into this recently and the solution* is to use papaja::apa_table
to output the tables.
https://www.rdocumentation.org/packages/papaja/versions/0.1.0.9997/topics/apa_table"

**Comment by jkosie** [comment]
I'm working on it. Just changing to papaja::apa_table doesn't seem to work.

**Comment by jkosie** [comment]
I'm reopening this issue - I've been working on this for a couple of hours and I still can't get it to work.

**Comment by kristabh** [comment]
I also spent quite a while on this yesterday with no luck.  Maybe for now we could find a temporary solution that would allow us to just copy-paste decently formatted tables into the Word version so I can submit. The Word version is only for the typesetting. Everything knits fine for .pdf.

**Comment by jkosie** [comment]
I think I can knit the appendix by itself and it looks okay. I'll try to just do that and paste things together this afternoon.

**Comment by kristabh** [comment]
Thanks so much for your help with this @jkosie !

**Comment by jkosie** [comment]
Here's an edited version of the Appendix. 

When I knit the rest of the paper to word, Table 1 (the landscape table) disappears and two other tables are formatted strangely. Does this happen for you? I thought I'd check before I keep playing around with it.

[mb1b_appendix.docx](https://github.com/manybabies/mb1b-analysis-public/files/5158734/mb1b_appendix.docx)

**Comment by kristabh** [comment]
Yes, that same thing happens to me.  I had not noticed it yet- was so focused on the Appendix. Thanks for continuing to work on this issue!  If the solution is to get these tables working manually so we can submit the .doc file, then I guess so be it...

**Comment by jkosie** [comment]
Just a note that I'm working on this. Hopefully will be finished soon. There are a lot of weird issues - for example, tables that use the exact same code, like for the matched and unmatched samples, print completely differently in the word doc.

**Comment by jkosie** [comment]
Okay, here's a word version of the paper, including the appendix.

I've edited the tables that were formatted strangely. The placement of the tables also changed a little from the pdf version, but I tried to copy/paste them into places that make sense. Table 1 is not landscape version in the word doc, but I reduced the font a little and I think it looks okay.

One other weird thing is that the numbers in the tables changed slightly from the pdf version printed with kable vs. the word version printed with papaja::apa_table. The differences are very small (e.g,. 0.0495 in the pdf becomes 0.0498 in the word doc) - seems likely to be just a difference in how the two table functions round things. The slight differences don't change any conclusions at all. However, I went through each table and compared the word version to the pdf version. When the pdf and word version differed, I manually changed the values to match the pdf. 

I ended up having to enter or edit a lot of values by hand. I've already checked and double-checked myself, but it might be good for someone to glance over the numbers one more time to check for errors.

[mb1b_paper.docx](https://github.com/manybabies/mb1b-analysis-public/files/5164579/mb1b_paper.docx)
